docs-sphinx/user-guide/how-to-use-header-only-layoutbuilder.md

```diff
@@ -0,0 +1,260 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: C++14
+  language: C++14
+  name: xcpp14
+---
+
+How to use the header-only LayoutBuilder in C++
+===============================================
+
```

Comment:
I think, the user guide should mention how to get appropriate `-I` and the compiler flag options available from `awkward._v2.config`:
```
yana@iannas-macbook-pro-2 awkward % python -m awkward._v2.config --cflags
-std=c++17 -I/Users/yana/Projects/PR1625/awkward/awkward/src/_v2/cpp-headers
yana@iannas-macbook-pro-2 awkward % python -m awkward._v2.config --cflags-only-I
-I/Users/yana/Projects/PR1625/awkward/awkward/src/_v2/cpp-headers
yana@iannas-macbook-pro-2 awkward % python -m awkward._v2.config --incdir
/Users/yana/Projects/PR1625/awkward/awkward/src/_v2/cpp-headers
```

Reply:
Can you please elaborate?

Reply:
Awkward Array can be installed [from PyPI](https://pypi.org/project/awkward) using pip:

`pip install awkward`

To get the compiler options needed to pick up the `LayoutBuilder` from this installation:
`python -m awkward._v2.config --cflags`

A user would need to pass these options to the compiler in order to use it.